lib/improver/spotdata/read_input.py

```diff
@@ -130,28 +138,42 @@ def get_additional_diagnostics(diagnostic_name, diagnostic_data_path,
     """
```

Comment:
I have made this change. I'm not going to move the configuration element outside of the routine yet, though I agree that may be appropriate as and when there are more methods,